src/pages/PostDetail/index.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  IonContent,
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonBackButton,
+  IonButtons,
+  IonCardSubtitle,
+  IonCardTitle,
+} from '@ionic/react';
+import React from 'react';
+import Interweave from 'interweave';
+import { RouteComponentProps } from 'react-router';
+import { useResource } from 'rest-hooks';
+import PostResource from '../../resources/post';
+
+type NewsDetailProps = RouteComponentProps<{
```

Comment:
Be wary about interchanging Posts and News.

Reply:
Fixed as per last commit